README.md

```diff
@@ -23,11 +23,50 @@ Type the path to the output to run the program.
 
 After running the program generates a file name TimeSheet.txt, this is where you copy the inputed hours from.
 
+## Sample run of the app
 
+- The week that starts with April 5 as a Sunday.
 
+```
+Prompt 1: Please enter Sundays month(Ex January - 1):
+Meaning: Please enter the month in numerical format for the Sunday of the week you worked on.
+Example input: 4 //(Because it's april)
+```
 
+```
+Prompt 2: Please enter Sundays, day of the month(Ex 15th - 15):
+Meaning: Please enter the date for sunday for the week.
+Example input: 5 //(Sunday for this week is 5th so 5).
+```
+```
+Prompt 3: How many time blocks did you work on Sunday:
+Meaning: A time block is a period of time for which you worked continuously in the day, you could've worked on multiple periods of time in a day, this is just a number of the blocks out of your day's schedule where you worked.
+Example input: 0 //(Assuming I didn't work on Sunday).
+```
 
+```
+Prompt 4: How many time blocks did you work on Monday:
+Meaning: This will continue for the rest of the days of week until Saturday. Now assuming you did work on Monday in two different time periods, you will have the following prompts.
+Example input: 2 //(Since I worked for two time blocks).
+```
 
 
+```
+Prompt 5: Monday start time for block - 1 (Ex 8:30 - 8.50):
+Meaning: Now we just enter the start time and then another prompt will ask us for hours worked for that time block and we don't need to worry about end time.
+Example input: 9 //(Your input format has a range of [9, 9.0, 9:00], it shoudln't matter)
```

Comment:
This looks solid but if they use 9:00 it may cause some issues.  The program reads the input as a float so the colon character is a problem.